### ISSUE TITLE
Segmentation fault error while trying to make a diff

### ISSUE THREAD

**Issue by conradojordan** [danielroedl/vsc-meld-diff#7]
Hello,

I had been using Meld Diff for quite a while, but after formatting my computer and upgrading my Ubuntu to 20.04, I have been getting a Segmentation Fault error when trying to use Meld to diff two files inside VSCode (using the "Select for Meld compare" feature).

Here is a small video showing the error:

https://user-images.githubusercontent.com/17773801/117676144-a874ef80-b183-11eb-911a-a09a4504f9fb.mp4

And what shows up in the Developer Tools when I open it:

![image](https://user-images.githubusercontent.com/17773801/117676754-3650da80-b184-11eb-86e7-e4ef10866cf9.png)

When I try to use `meld` command manually in a terminal it works perfectly (as shown in the video as well).

VSCode version is 1.56.0
Meld diff extension version is 1.1.0
Ubuntu version 20.04 (don't know if it matters)

`meld` is in PATH and output of `which meld` is `/usr/bin/meld`

**Comment by danielroedl** [comment]
Hello,
I have tried to reproduce your error but in my test environment everthing runs fine.
I have used a fresh installed Ubuntu 20.04, with the same vscode version. I installed meld with apt.
I also cannot found a reason from source code side, why this don't work on your system.

Can you please try to open the settings of melddiff and enter `/usr/bin/meld` instead of only `meld` in the `Diff Command` setting?

**Comment by conradojordan** [comment]
Hello Daniel,

I had tried changing the command setting for `/usr/bin/meld` but it also didn't work. Also tried reinstalling both meld - via apt and the software store - and the meld diff extension to no avail.

Well, I have tried reinstalling VSCode now and it worked (and feel a bit dumb for not trying this before). I will be closing this issue, but thanks for the attention and your time.

I would also like you to know that I really appreciate this extension (and know other developers that do too), so thank you for your effort in developing/maintaining it. Have a good one!
